app/views/themes/neutral_repository/hyrax/homepage/_recent_document.html.erb

```diff
@@ -1,8 +1,7 @@
-<%# OVERRIDE Hyrax v5.0.0rc2 template for client theming and shared search %>
 <h3 class="sr-only"><%= t('hyrax.homepage.recently_uploaded.document.title_label') %></h3>
 <div class="recently-uploaded pb-3 col-6 col-md-6">
   <%= render_thumbnail_tag recent_document, {class: 'img-fluid mx-auto d-block'}, {suppress_link: true} %>
   <div class="recent-doc-title">
     <%= link_to recent_document.title_or_label, generate_work_url(recent_document, request) %>
   </div>
-</div>
+<% end %>
```

Comment:
same issue as app/views/themes/institutional_repository/hyrax/homepage/_recent_document.html.erb

Reply:
this is my fault. i got this wrong with the merge conflict and am trying to resolve it.